neo/Prompt/Commands/Config.py

```diff
@@ -53,23 +53,23 @@ def __init__(self):
         super().__init__()
 
     def execute(self, arguments):
-        c1 = get_arg(arguments)
-        if c1 is not None:
-            c1 = c1.lower()
-            if c1 == 'on' or c1 == '1':
-                settings.set_log_smart_contract_events(True)
-                print("Smart contract event logging is now enabled")
-                return c1
-            elif c1 == 'off' or c1 == '0':
-                settings.set_log_smart_contract_events(False)
-                print("Smart contract event logging is now disabled")
-                return c1
-            else:
-                print("Cannot configure log. Please specify on|off or 1|0")
-                return
+        if len(arguments) == 0:
+            print("Please specify the required parameter")
+            return False
+
+        try:
+            flag = bool(util.strtobool(arguments[0]))
+            settings.set_log_smart_contract_events(flag)
+        except ValueError:
+            print("Invalid option")
+            return False
+
+        if flag:
+            print("Smart contract event logging is now enabled")
+            return True
         else:
-            print("Cannot configure log. Please specify on|off or 1|0")
-            return
+            print("Smart contract event logging is now disabled")
+            return False
```

Comment:
Referring to follow-on commands, I think this should be `return True` since the command is successfully disabling the SC event logging. See https://github.com/CityOfZion/neo-python/pull/783/files#diff-799c82d84fa184b364281a4e3f88141cR100 and https://github.com/CityOfZion/neo-python/pull/783/files#diff-799c82d84fa184b364281a4e3f88141cR128

Reply:
good catch! 💪